stable/external-dns/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: external-dns
-version: 2.0.2
-appVersion: 0.5.15
+version: 2.0.3
+appVersion: 0.5.16
```

Comment:
You don't need to bump the `appVersion`. Indeed, we haven't release the version `0.5.16` yet, see https://hub.docker.com/r/bitnami/external-dns/tags/